trompace/mutations/controlaction.py

```diff
@@ -137,3 +137,14 @@ def mutation_request_controlaction(controlaction_id: str, entrypoint_id: str, pr
         }
     }
     return format_mutation("RequestControlAction", args)
+
+def mutation_update_controlaction(controlaction_id: str, actionStatus: trompace.StringConstant):
+    args = {
+       "identifier": controlaction_id,
+       "actionStatus": actionStatus
+    }
+    return format_mutation("UpdateControlAction", args)
+
+def mutation_addactioninterfance_result(controlaction_id:str, thing_id:str):
```

Comment:
same as the last one, add some documentation
naming - actioninterface - not actioninterfance

Take a look at the messages reported by the pep8 bot, there are a few styling things that you can improve. What editor do you use? I can help you configure some formatting tools to make this better if you want.